tests/glsl-intrinsic/shader-subgroup/shader-subgroup-arithmetic.slang

```diff
@@ -0,0 +1,200 @@
+// not testing glsl or spirv due to crashing issue from some odd code emitting which looks right but fails
+//TEST:SIMPLE(filecheck=CHECK_GLSL):  -allow-glsl -stage compute -entry computeMain -target glsl
+//TEST:SIMPLE(filecheck=CHECK_SPV):  -allow-glsl -stage compute -entry computeMain -target spirv
+//TEST:SIMPLE(filecheck=CHECK_HLSL): -allow-glsl -stage compute -entry computeMain -target hlsl
+//TEST:SIMPLE(filecheck=CHECK_CUDA): -allow-glsl -stage compute -entry computeMain -target cuda 
+// not testing cpp due to missing impl
+// TEST:SIMPLE(filecheck=CHECK_CPP):  -allow-glsl -stage compute -entry computeMain -target cpp
```

Comment:
My personal preference of how to disabling test here is to use
```
//T-EST
```
Just a white space character works but little hard to see.